doc/oomd/SONiC OOM daemon.md

```diff
@@ -0,0 +1,255 @@
+# SONiC OOM daemon
+
+# Table of Contents
+- [Table of Contents](#table-of-contents)
+- [About this Manual](#about-this-manual)
+- [1 Functional Requirements](#1-functional-requirement)
+- [2 Configuration and Management Requirements](#2-configuration-and-management-requirements)
+  * [2.1 SONiC CLI](#21-sonic-cli)
+  * [2.2 Config DB](#22-config-db)
+- [3 Design](#design)
+  * [3.1 OOMD Implementation](#31-oomd-Implementation)
+  * [3.2 Default OOMD config](#32-default-oomd-config)
+  * [3.3 ConfigDB Schema](#34-configdb-schema)
+  * [3.4 CLI](#35-cli)
+- [4 Error handling](#error-handling)
+- [5 Serviceability and Debug](#serviceability-and-debug)
+- [6 Unit Test](#unit-test)
+- [8 References](#references)
+
+
+# About this Manual
+This document provides a detailed description on the new features for:
+ - Protect SONiC system memory by OOM daemon.
+ - OOM daemon config command.
+ - OOM daemon high level design.
+ - OOM daemon ConfigDB schema
+
+### SONiC memory issue sloved by this feature.
```

Comment:
>SONiC memory issue [](http://example.com/codeflow?start=4&length=18)

This section is duplicated with the other markdown file. Could you organized the two design in a hierarchy and merge the similar content?

Reply:
Fixed, as discussion, add a reference in the login session limit document.